### PR TITLE
Remove nbdev from peptdeep

### PR DESCRIPTION
**Pull request by jalew188**
Remove nbdev-related files

### CODE REVIEW

**Review by mschwoer — APPROVED** [review]
LGTM!

File: requirements/requirements_development.txt [hunk 1]
```diff
@@ -8,7 +8,6 @@ pipdeptree
 ipykernel
 tqdm
 wget
-nbdev
```
[the remark this answers is not in the document]
Comment (jalew188) [reply requirements/requirements_development.txt:11]: Will do

**Review by GeorgWa — APPROVED** [review]
LGTM👍

File: requirements/requirements_development.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (jalew188) [reply requirements/requirements_development.txt:11]: The same for alphabase/alpharaw

**Pull request merged**